atomics/T1069.002/T1069.002.yaml

```diff
@@ -242,4 +242,43 @@ atomic_tests:
       ldifde.exe -f #{output_path}\#{output_file} -p subtree
     cleanup_command: |
       del #{output_path}\#{output_file}
-    name: command_prompt
\ No newline at end of file
+    name: command_prompt
+- name: Active Directory Domain Search Using LDAP
+  description: |
+    Output information from LDAPSearch. LDAP Password is the admin-user password on Active Directory
+  supported_platforms:
+  - linux
+  input_arguments:
+    domain:
+      description: The domain to be tested
+      type: string
+      default: example
+    top_level_domain:
+      description: The top level domain (.com, .test, .remote, etc... following domain, minus the .)
+      type: string
+      default: com
+    user:
+      description: username@domain of a user
+      type: string
+      default: user@example.com
+    password:
+      description: password of the user referenced inside user
+      type: string
+      default: s3CurePssw0rD!
+    domain_controller:
+      description: Name of the domain_controller machine, defined in etc/hosts
+      type: string
+      default: adhostname
+  dependency_executor_name: sh
+  dependencies:
+  - description: |
+      Packages sssd-ad sssd-tools realmd adcli installed and realm available, ldapsearch
+    prereq_command: |
+      which ldapsearch
+    get_prereq_command: |
+      echo missing ldapsearch command; exit 1
+  executor:
+    elevation_required: false
+    command: |
+      ldapsearch -h #{domain_controller} -p 389 -x -D #{user} -w #{password} -b "CN=Users,DC=#{domain},DC=#{top_level_domain}" "(objectClass=group)" -s sub -a always -z 1000 dn 
```

Comment:
Can you mention which linux distribution which you are using ? I tried with centos and ubuntu. I am getting an error 
Unknown option for -h and -p as well

Reply:
Ah, looks like -h and -p are deprecated for -H instead. I'll do a quick rewrite.

I used Ubuntu, would that be placed in the description?

Reply:
No worries. I just wanted to confirm so that I can run atomics on it. Thanks again.

Reply:
Hey, just made that change from -h -p to -H. Sorry it took a while, got caught up in other issues. Mind taking a look?

Thanks